dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -897,6 +897,34 @@ def generate_teacher_student(name, teacher_authorized)
   }
 end
 
+And /^I create a new section with course "([^"]*)"(?: and unit "([^"]*)")?$/ do |primary, secondary|
+  individual_steps %Q{
+    When I press the new section button
+    Then I should see the new section dialog
+
+    When I select email login
+    Then I wait to see "#uitest-primary-assignment"
+  }
+
+  steps %Q{
+    When I select the "#{primary}" option in dropdown "uitest-primary-assignment"
+  }
+
+  if secondary
+    individual_steps %Q{
+      And I wait to see "#uitest-secondary-assignment"
+      And I select the "#{secondary}" option in dropdown "uitest-secondary-assignment"
+    }
+  end
+
+  individual_steps %Q{
+    And I scroll the save button into view
```

Comment:
I don't think this step is necessary anymore - there should be a fixed footer on that dialog now, so no scrolling is needed.